Content.Server/Discord/Managers/LastMessageWebhookManager.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Server.Discord.Managers
+{
+    public class LastMessageWebhookManager
+    {
+        public const int MaxMessageSize = 2000; // CAN'T BE MORE THAN 2000! DISCORD LIMIT.
+        private const int MaxMessagesPerBatch = 15;
+        private const int MessageDelayMilliseconds = 2000;
+
+        /// <summary>
+        ///     Sends a list of messages to a specified webhook, handling rate limiting.
+        /// </summary>
+        /// <param name="webhookId">The identifier of the webhook to send messages to.</param>
+        /// <param name="messages">The list of messages to be sent.</param>
+        /// <returns>A task representing the asynchronous operation.  :nerd:</returns>
+        public async Task SendMessagesAsync(WebhookIdentifier webhookId, List<string> messages)
+        {
+            var discordWebhook = new DiscordWebhook();
+            int messageCount = 0;
```

Comment:
You shouldn't be constructing the webhook every time, just persist it.

Reply:
Done.